### PR TITLE
Fix client connection

### PR DESCRIPTION
**Pull request by Karang**
Signed-off-by: Karang arthur.hennequin@free.fr

### CODE REVIEW

**Comment by RoyAwesome** [comment]
Can you split this into two pull requests? I want to pull the tickstage stuff but not the texture stuff (I have a better idea for this)

**Comment by Karang** [comment]
Hum, what's your idea? We need this feature for our plugin. Tell me, I'll implement it in Spout.

**Comment by Karang** [comment]
@RoyAwesome I reverted the commit you don't wanted, but tell me your idea I need the feature ASAP
Edit: i merged the commits into one

File: src/main/java/org/spout/engine/protocol/SpoutClientSession.java [hunk 1]
```diff
@@ -57,6 +57,8 @@ public SpoutClientSession(SpoutClient engine, Channel channel, Protocol bootstra
 	public void send(boolean upstream, boolean force, Message message) {
 		if (!upstream) {
 			getEngine().getLogger().warning("Attempt made to send packet to client");
+		} else {
+			getEngine().getLogger().warning("Attempt made to send packet to server");
```
Comment (zml2008) [inline src/main/java/org/spout/engine/protocol/SpoutClientSession.java:61]: uh no, this will be logged every packet.

**Pull request merged**